section-05-production-model-package/regression_model/config.yml

```diff
@@ -0,0 +1,172 @@
+# Package Overview
+package_name: regression_model
+
+# Data Files
+training_data_file: train.csv
+test_data_file: test.csv
+
+# Variables
+# The variable we are attempting to predict (sale price)
+target: SalePrice
+
+pipeline_name: regression_model
+pipeline_save_file: regression_model_output_v
+
+# Will cause syntax errors since they begin with numbers
+variables_to_rename:
+  1stFlrSF: FirstFlrSF
+  2ndFlrSF: SecondFlrSF
+  3SsnPorch: ThreeSsnPortch
+
+features:
+  - MSSubClass
+  - MSZoning
+  - LotFrontage
+  - LotShape
+  - LandContour
+  - LotConfig
+  - Neighborhood
+  - OverallQual
+  - OverallCond
+  - YearRemodAdd
+  - RoofStyle
+  - Exterior1st
+  - ExterQual
+  - Foundation
+  - BsmtQual
+  - BsmtExposure
+  - BsmtFinType1
+  - HeatingQC
+  - CentralAir
+  - FirstFlrSF  # renamed
+  - SecondFlrSF  # renamed
+  - GrLivArea
+  - BsmtFullBath
+  - HalfBath
+  - KitchenQual
+  - TotRmsAbvGrd
+  - Functional
+  - Fireplaces
+  - FireplaceQu
+  - GarageFinish
+  - GarageCars
+  - GarageArea
+  - PavedDrive
+  - WoodDeckSF
+  - ScreenPorch
+  - SaleCondition
+  # this one is only to calculate temporal variable:
+  - YrSold
+
+# set train/test split
+test_size: 0.1
+
+# to set the random seed
+random_state: 0
+
+alpha: 0.001
+
+# categorical variables with NA in train set
+categorical_vars_with_na_frequent:
+  - BsmtQual
+  - BsmtExposure
+  - BsmtFinType1
+  - GarageFinish
+
+categorical_vars_with_na_missing:
+  - FireplaceQu
+
+numerical_vars_with_na:
+  - LotFrontage
+
+temporal_vars:
+  - YearRemodAdd
+
+ref_var: YrSold
+
+# this variable is to calculate the temporal variable,
+# can be dropped afterwards
+drop_features:
+  - YrSold
+
```

Comment:
why here with - and above right after the : ?

I would remove lines 87-90, in the pipeline (unless you changed it), I am using `ref_var` from the 2 transformers that need that variable. So drop_features is not used.

Reply:
This syntax is how you define a list in yaml. Removing the `drop_features` as you suggest